recipes/ceres-solver/all/conanfile.py

```diff
@@ -73,55 +66,96 @@ def configure(self):
             raise ConanInvalidConfiguration("Ceres-solver only links against the release version of glog")
         if self.options.use_glog and not self.options.use_gflags: #At this stage we can't check the value of self.options["glog"].with_gflags so we asume it is true because is the default value
             raise ConanInvalidConfiguration("To depend on glog built with gflags (Default behavior) set use_gflags=True, otherwise Ceres may fail to link due to missing gflags symbols.")
+        if tools.Version(self.version) >= "2.0":
+            # 1.x uses ceres-solver specific FindXXX.cmake modules
+            self.generators.append("cmake_find_package")
+            if self.settings.compiler.cppstd:
+                tools.check_min_cppstd(self, 14)
+            self._check_cxx14_supported()
 
     def requirements(self):
-        self.requires("eigen/3.3.7")
+        self.requires("eigen/3.3.8")
         if self.options.use_glog:
             self.requires("glog/0.4.0")
         if self.options.use_gflags:
             self.requires("gflags/2.2.2")
             self.options["gflags"].nothreads = False
         if self.options.use_TBB:
-            self.requires("tbb/2020.0")
+            self.requires("tbb/2020.2")
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         extracted_dir = self.name + "-" + self.version
         os.rename(extracted_dir, self._source_subfolder)
 
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)       #You can check what these flags do in http://ceres-solver.org/installation.html
+        self._cmake.definitions["CMAKE_CXX_STANDARD"] = "14"
```

Comment:
hmm this was not required for V1, and V2 will through if it lower... is this still required?

Reply:
I don't remember why I added this. Probably for troubleshooting.